src/WinRT.Runtime/Projections/PropertyChangedEventArgs.cs

```diff
@@ -86,7 +91,78 @@ public unsafe ObjectReferenceValue CreateInstance(string name)
                 MarshalInspectable<object>.DisposeAbi(__innerInterface);
             }
         }
+    }
+
+
+
+    [global::WinRT.ObjectReferenceWrapper(nameof(_obj))]
```

Comment:
This attribute should only be used downlevel. On .NET, shouldn't we also implement `IWinRTObject` here?